README.md

```diff
@@ -209,44 +209,45 @@ oc apply -f deploy/olm-catalog/subscription.yaml
 ```
 
 ### Creating a Windows MachineSet
-Below is the example of a Windows MachineSet which can create Windows Machines that the WMCO can react upon. Please 
+Below is the example of an AWS Windows MachineSet which can create Windows Machines that the WMCO can react upon. Please
 note that the `windows-user-data` secret will be created by the WMCO lazily when it is configuring the first Windows 
 Machine. After that, the `windows-user-data` will be available for the subsequent MachineSets to be consumed. It might 
 take around 10 minutes for the Windows VM to be configured so that it joins the cluster. Please note that the MachineSet
-should have `machine.openshift.io/os-id: Windows` label and the image should point to a Windows image with a container
-run-time installed.
-
-In order to get the `infrastructureID` use:
+should have `machine.openshift.io/os-id: Windows` label, and _image_ should point to a Windows image with a container
```

Comment:
`image` -> `<windows_image_with_container_runtime_installed>`.

Reply:
<windows_image_with_container_runtime_installed> can also be replaced with something shorter, but we should make sure that the instructions and the template match.

Reply:
Please suggest something shorter and I will make the change.

Reply:
<windows_containers_ami>?